Makefile

```diff
@@ -33,7 +33,7 @@ CONTAINER_BUILD_PARAMS = --network=host --label git_revision=${GIT_REVISION} ${C
 # RHCOS_VERSION should be consistent with BaseObjectName in pkg/s3wrapper/client.go
 RHCOS_VERSION := $(or ${RHCOS_VERSION},46.82.202009222340-0)
 BASE_OS_IMAGE := $(or ${BASE_OS_IMAGE},https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/${RHCOS_VERSION}/x86_64/rhcos-${RHCOS_VERSION}-live.x86_64.iso)
-OPENSHIFT_INSTALL_RELEASE_IMAGE := $(or ${OPENSHIFT_INSTALL_RELEASE_IMAGE},quay.io/openshift-release-dev/ocp-release:4.6.4-x86_64)
+OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, '{"4.6": {"release": "quay.io/openshift-release-dev/ocp-release:4.6.4-x86_64"}}')
```

Comment:
maybe we should stop working with environment variables in the makefile and start working with configuration files? it fills like it's getting out of hand, probably not for this PR

Reply:
I agree. It's really a labyrinth over there. 
We should do that on a separate PR.